packages/flutter/test/widgets/actions_test.dart

```diff
@@ -1009,8 +1009,10 @@ void main() {
         .toList();
 
       expect(description.length, equals(2));
-      expect(description[0], equalsIgnoringHashCodes('dispatcher: ActionDispatcher#00000'));
-      expect(description[1], equals('actions: {}'));
+      expect(
+        description,
+        equalsIgnoringHashCodes(<String>['dispatcher: ActionDispatcher#00000', 'actions: {}']),
```

Comment:
nit: I think these read better if each entry in the list is given their own line.

Reply:
Done.